src/pudl/workspace/resource_cache.py

```diff
@@ -106,7 +106,8 @@ def add(self, resource: PudlResourceKey, content: bytes):
             return
         path = self._resource_path(resource)
         path.parent.mkdir(parents=True, exist_ok=True)
-        path.open("wb").write(content)
+        with path.open("wb") as file:
+            file.write(content)
```

Comment:
This was generating a warning about the file resource being left open, so I switched to a context manager.